submissions/VolodymyrRutskyi/task_js_dom/main.js

```diff
@@ -0,0 +1,186 @@
+let data = [
```

Comment:
Must be `const`. Rename variable. They must have more informative naming. (`methodList` for example)

Reply:
Fixed